### PR TITLE
chore: updated docs according to the PR

### PR DESCRIPTION
**Pull request by Mikelle**
Waiting for that to go live: https://github.com/primev/mev-commit/pull/274

### CODE REVIEW

File: concepts/bids/bidder-deposit.mdx [hunk 1]
```diff
@@ -24,7 +24,7 @@ Since deposits are split across $\text{blocksPerWindow} = 10$ L1 blocks, bidders
 
 ## Withdrawal Time
 
-The earliest withdrawal time can be calculated as the withdrawal start time, n-2 windows which is 30 L1 blocks (360 seconds) and oracle lag which is 20 L1 blocks (240 seconds) totalling 600 seconds, or 10 minutes. Therefore, the total time before funds can be withdrawn is approximately 50 blocks (10 minutes) from the initial deposit.
+The earliest withdrawal time can be calculated as the withdrawal start time, n-2 windows which is 30 L1 blocks (360 seconds) and oracle lag which is 10 L1 blocks (120 seconds) totalling 480 seconds, or 10 minutes. Therefore, the total time before funds can be withdrawn is approximately 40 blocks (8 minutes) from the initial deposit.
```
Comment (chrmatt) [inline concepts/bids/bidder-deposit.mdx:27]: The first sentence is confusing. Also, the 480 seconds should be 8 minutes instead of 10.

**Review by chrmatt — CHANGES_REQUESTED** [review]
I suggest to also mention the auto deposit feature and its implications (30x deposit instead of 10x) in the bidder-deposit.mdx since this is important to know and should not be buried inside the node commands. Maybe just copy/paste the introductory text with the Note from the bidder-node-commands.

File: concepts/bids/bidder-deposit.mdx (hunk 1 above)
Comment (Mikelle) [reply concepts/bids/bidder-deposit.mdx]: Thank you, fixed!

**Review by chrmatt — APPROVED** [review]
Looks good. I've added another sentence to clarify the autodeposits in the beginning.

**Pull request merged**